src/components/Text/Text.stories.tsx

```diff
@@ -173,3 +184,90 @@ export const TextVariantInherit: StoryObj<Args> = {
     variant: 'inherit',
   },
 };
+
+const TextPassageTemplate = (args: Args) => (
```

Comment:
Replaced `TextPassage` with `Text`, surprisingly only one visual diff (noted above), but let me know if the `TextPassage` component shouldn't be deleted